tcf_website/templates/reviews/new.html

```diff
@@ -4,75 +4,156 @@
+            </select>
           </div>
         </div>
-        <div class="form-group mb-3">
-          <label for="reviewText">Tell us more about the course & instructor:</label>
-          <textarea class="form-control" id="reviewText" name="reviewText" placeholder="e.g. I love CS 2150, Bloomfield is great!"></textarea>
+        <hr>
+
+        <!-- Review written content -->
+        <div class="new-review-content form-row mb-3">
+          <h4 class="review-form-header form-group col-12 my-4">
+            Tell us more about the course and instructor</h4>
+          <div class="form-group col mb-3">
+            <textarea class="form-control" id="reviewText" name="reviewText" rows="5"
+                      placeholder="e.g. I love CS 2150, Bloomfield is great!"></textarea>
+          </div>
         </div>
-        <div class="form-row d-flex justify-content-space-around mb-3">
-          <div class="form-group col mr-5">
-                <div class="mt-3">
-                    <label for="instructorRating">Instructor rating</label>
-                    <div class="d-flex">
-                        <span class="mr-2">1</span>
-                        <input type="range" class="custom-range" min="1" max="5" step="1" id="instructorRating" name="instructorRating" required>
-                        <span class="ml-2">5</span>
-                    </div>
-                </div>
-                <div class="mt-3">
-                    <label for="difficulty">Difficulty</label>
-                    <div class="d-flex">
-                        <span class="mr-2">1</span>
-                        <input type="range" class="custom-range" min="1" max="5" step="1" id="difficulty" name="difficulty" required>
-                        <span class="ml-2">5</span>
-                    </div>
-                </div>
-                <div class="mt-3">
-                    <label for="recommendability">Recommendability</label>
-                    <div class="d-flex">
-                        <span class="mr-2">1</span>
-                        <input type="range" class="custom-range" min="1" max="5" step="1" id="recommendability" name="recommendability" required>
-                        <span class="ml-2">5</span>
-                    </div>
-                </div>
+        <hr>
+
+        <!-- Review ratings for course -->
+        <div class="new-review-ratings form-row mb-3">
+          <div class="form-group col-sm-6 pr-sm-5">
+              <h4 class="review-form-header col-12 p-0 my-4">
+                Rate the course</h4>
+              <div class="mt-3">
+                  <label for="instructorRating" class="required-field">
+                    <i class="fas fa-user"></i> Instructor
+                  </label>
+                  <div class="d-flex">
+                      <span class="mr-2">1</span>
+                      <input type="range" class="custom-range" min="1" max="5" step="1"
```

Comment:
Not sure if this will break any styling you used, but for Bootstrap you generally include `class="form-control"` for some default styles
https://getbootstrap.com/docs/4.0/components/forms/#form-controls
Edit: probably `form-control-range` in this case https://getbootstrap.com/docs/4.1/components/forms/#range-inputs

Reply:
Oh also this isn't essential but it'd be cool to have the slider handle be orange

Reply:
I'll play around with it!

Reply:
Got it to work :)
![image](https://user-images.githubusercontent.com/4724192/100167002-02ab1e00-2e8c-11eb-98fa-9463d8360e6f.png)

Reply:
I don't think the `form-control` or `form-control-range` looks good lol...
![image](https://user-images.githubusercontent.com/4724192/100167071-38500700-2e8c-11eb-96fa-7261ddaa56a6.png)

Reply:
hmm idk

Reply:
![image](https://user-images.githubusercontent.com/4724192/100183284-837a1200-2eac-11eb-9530-4871a1894c29.png)

This is with form-control-range

Reply:
I think that looks good but do what you prefer

Reply:
I like the fill-in but I don't think the style (thin, outlined) matches the rest of the form
Also i'm unable to get the color to change with the same CSS
Overall we'd have to overwrite a lot of styling to get it to work and I don't think it's worth looking into currently